dist/index.html

```diff
@@ -4,6 +4,7 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link rel="stylesheet" type="text/css" href="styles.css" />
     <title>SeadFood Restaurant</title>
   </head>
   <body>
```

Comment:
You should have`<div>` with `id=content` in this file. more details above....

Reply:
Sure, i forget to use it and only use my script file to load everything included the content div. thanks for the reminder